src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
My knee-jerk reaction to this change is that why introducing a new CSS class instead of putting `content: ''` to `.bx--breadcrumb-item:last-child::after`.

The PR description seems to imply that @rahulrkr08 has a reason behind it, but not sure what specific problem he/she is trying to resolve with a new class (instead of existing pseudo). Having an example or two would help us understanding the intent.